types/vote_set_test.go

```diff
@@ -210,38 +215,38 @@ func TestVoteSet_2_3MajorityRedux(t *testing.T) {
 		vote := withValidator(voteProto, adrr, 66)
 		_, err = signAddVote(privValidators[66], withBlockHash(vote, nil), voteSet)
 		require.NoError(t, err)
-		blockID, ok = voteSet.TwoThirdsMajority()
+		blockID, _, ok = voteSet.TwoThirdsMajority()
 		assert.False(t, ok || !blockID.IsZero(),
 			"there should be no 2/3 majority: last vote added was nil")
 	}
 
-	// 68th validator voted for a different BlockParts PartSetHeader
-	{
-		pubKey, err := privValidators[67].GetPubKey()
-		require.NoError(t, err)
-		addr := pubKey.Address()
-		vote := withValidator(voteProto, addr, 67)
-		blockPartsHeader := PartSetHeader{blockPartsTotal, crypto.CRandBytes(32)}
-		_, err = signAddVote(privValidators[67], withBlockPartSetHeader(vote, blockPartsHeader), voteSet)
-		require.NoError(t, err)
-		blockID, ok = voteSet.TwoThirdsMajority()
-		assert.False(t, ok || !blockID.IsZero(),
-			"there should be no 2/3 majority: last vote added had different PartSetHeader Hash")
-	}
+	// // 68th validator voted for a different BlockParts PartSetHeader
+	// {
+	// 	pubKey, err := privValidators[67].GetPubKey()
+	// 	require.NoError(t, err)
+	// 	addr := pubKey.Address()
+	// 	vote := withValidator(voteProto, addr, 67)
+	// 	blockPartsHeader := PartSetHeader{blockPartsTotal, crypto.CRandBytes(32)}
+	// 	_, err = signAddVote(privValidators[67], withBlockPartSetHeader(vote, blockPartsHeader), voteSet)
+	// 	require.NoError(t, err)
+	// 	blockID, _, ok = voteSet.TwoThirdsMajority()
+	// 	assert.False(t, ok || !blockID.IsZero(),
+	// 		"there should be no 2/3 majority: last vote added had different PartSetHeader Hash")
+	// }
 
 	// 69th validator voted for different BlockParts Total
-	{
-		pubKey, err := privValidators[68].GetPubKey()
-		require.NoError(t, err)
-		addr := pubKey.Address()
-		vote := withValidator(voteProto, addr, 68)
-		blockPartsHeader := PartSetHeader{blockPartsTotal + 1, blockPartSetHeader.Hash}
-		_, err = signAddVote(privValidators[68], withBlockPartSetHeader(vote, blockPartsHeader), voteSet)
-		require.NoError(t, err)
-		blockID, ok = voteSet.TwoThirdsMajority()
-		assert.False(t, ok || !blockID.IsZero(),
-			"there should be no 2/3 majority: last vote added had different PartSetHeader Total")
-	}
+	// {
+	// 	pubKey, err := privValidators[68].GetPubKey()
+	// 	require.NoError(t, err)
+	// 	addr := pubKey.Address()
+	// 	vote := withValidator(voteProto, addr, 68)
+	// 	blockPartsHeader := PartSetHeader{blockPartsTotal + 1, blockPartSetHeader.Hash}
+	// 	_, err = signAddVote(privValidators[68], withBlockPartSetHeader(vote, blockPartsHeader), voteSet)
+	// 	require.NoError(t, err)
+	// 	blockID, _, ok = voteSet.TwoThirdsMajority()
+	// 	assert.False(t, ok || !blockID.IsZero(),
+	// 		"there should be no 2/3 majority: last vote added had different PartSetHeader Total")
```

Comment:
these tests were commented out instead of deleted because I wanted to make sure that this is okay. They do fail, but only because they artificially change the partsetheader. The partsetheader is included in header hash, so the blockID wouldn't match if this were the case.

Reply:
I am ok to delete those

Reply:
deleted here https://github.com/celestiaorg/lazyledger-core/pull/441/commits/daf469ae357c92674630e0b5c958f1ced4d4a8f5